static/app/views/replays/filters.tsx

```diff
@@ -2,13 +2,14 @@ import styled from '@emotion/styled';
 
 import DatePageFilter from 'sentry/components/datePageFilter';
 import EnvironmentPageFilter from 'sentry/components/environmentPageFilter';
-import SearchBar from 'sentry/components/events/searchBar';
 import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
 import ProjectPageFilter from 'sentry/components/projectPageFilter';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 import {Organization} from 'sentry/types';
 
+import SearchBar from './searchBar';
+
 type Props = {
```

Comment:
can we use full imports. I find it much easier to grep for places that import a bit of code when all the callsites spell things the same way. I'm going to get the linter changed too

```suggestion
import {Organization} from 'sentry/types';
import SearchBar from 'sentry/views/replays/searchBar';

type Props = {
```